### PR TITLE
EZEE-2572 Page Builder doesn't work with Map\Host matcher when SiteAccesses are configured for different domains

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZEE-2572
| Versions      | all

**Merge after EZEE 2.4.2 is out.**
 
Documented bug fix: Currently when the Page Builder doesn't show page under SiteAccesses which are matched using {{Map\Host}} matcher.

### CODE REVIEW

File: docs/releases/ez_platform_v2.4.md [hunk 1]
```diff
@@ -160,3 +160,69 @@ You are now able to load multiple Locations at once. The biggest benefit of this
...
 | [List of changes for rc1 of eZ Platform v2.4.0 on Github](https://github.com/ezsystems/ezplatform/releases/tag/v2.4.0-rc1) | [List of changes for rc1 for eZ Platform Enterprise Edition v2.4.0 on Github](https://github.com/ezsystems/ezplatform-ee/releases/tag/v2.4.0-rc1) |
 | [List of changes for beta1 of eZ Platform v2.4.0 on Github](https://github.com/ezsystems/ezplatform/releases/tag/v2.4.0-beta1) | [List of changes for beta1 of eZ Platform Enterprise Edition v2.4.0 on Github](https://github.com/ezsystems/ezplatform-ee/releases/tag/v2.4.0-beta1) |
+
+## eZ Platform v2.4.2
+
+!!! enterprise
+
+    ### Update eZ Enterprise 2.4 to 2.4.2
+    
+    This update fixes: **[EZEE-2572: Page Builder doesn't work with Map\Host matcher when SiteAaccess are configured for different domains.](https://jira.ez.no/browse/EZEE-2572)**
```
Comment (DominikaK) [inline docs/releases/ez_platform_v2.4.md]:
```suggestion
    This update fixes: **[EZEE-2572: Page Builder doesn't work with Map\Host matcher when SiteAccesses are configured for different domains.](https://jira.ez.no/browse/EZEE-2572)**
```

File: docs/releases/ez_platform_v2.4.md [hunk 2]
```diff
@@ -160,3 +160,69 @@ You are now able to load multiple Locations at once. The biggest benefit of this
...
+
+    Token based authentication for PB is enabled by default, however, the following steps are required during upgrade from v2.4 to v2.4.2+ Enterprise installation :
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/releases/ez_platform_v2.4.md]: Maybe `is enabled by default in v2.4.2`, if that's the case?

File: docs/releases/ez_platform_v2.4.md [hunk 3]
```diff
@@ -160,3 +160,70 @@ You are now able to load multiple Locations at once. The biggest benefit of this
...
+             ezpublish_front:
+                 # ...
+                 simple_preauth:
+                     authenticator: 'EzSystems\EzPlatformPageBuilder\Security\EditorialMode\TokenAuthenticator'
+                 form_login:
+                     require_previous_session: false
+                 # ...
+    ```
+     
+    4\. Add `page_builder.token_authenticator.enabled: true` to `/app/config/parameters.yml`:
```
Comment (micszo) [inline docs/releases/ez_platform_v2.4.md]: It appears this setting is already present in `app/config/config.yml`. :) (in 2.4.2-rc1)

File: docs/releases/ez_platform_v2.4.md [hunk 4]
```diff
@@ -160,3 +160,72 @@ You are now able to load multiple Locations at once. The biggest benefit of this
...
+             ezpublish_front:
+                 # ...
+                 simple_preauth:
+                     authenticator: 'EzSystems\EzPlatformPageBuilder\Security\EditorialMode\TokenAuthenticator'
+                 form_login:
+                     require_previous_session: false
+                 # ...
+    ```
+     
+    4\. Add parameter `page_builder.token_authenticator.enabled: true` to `/app/config/config/parameters.yml`:
```
Comment (DominikaK) [inline docs/releases/ez_platform_v2.4.md]:
```suggestion
    4\. Add parameter `page_builder.token_authenticator.enabled: true` to `/app/config/config.yml`:
```

Typo in the path, I think?

File: docs/releases/ez_platform_v2.4.md [hunk 5]
```diff
@@ -160,3 +160,72 @@ You are now able to load multiple Locations at once. The biggest benefit of this
 | [List of changes for final of eZ Platform v2.4.0 on Github](https://github.com/ezsystems/ezplatform/releases/tag/v2.4.0) | [List of changes for final for eZ Platform Enterprise Edition v2.4.0 on Github](https://github.com/ezsystems/ezplatform-ee/releases/tag/v2.4.0) |
 | [List of changes for rc1 of eZ Platform v2.4.0 on Github](https://github.com/ezsystems/ezplatform/releases/tag/v2.4.0-rc1) | [List of changes for rc1 for eZ Platform Enterprise Edition v2.4.0 on Github](https://github.com/ezsystems/ezplatform-ee/releases/tag/v2.4.0-rc1) |
 | [List of changes for beta1 of eZ Platform v2.4.0 on Github](https://github.com/ezsystems/ezplatform/releases/tag/v2.4.0-beta1) | [List of changes for beta1 of eZ Platform Enterprise Edition v2.4.0 on Github](https://github.com/ezsystems/ezplatform-ee/releases/tag/v2.4.0-beta1) |
+
+## eZ Platform v2.4.2
+
+!!! enterprise
+
+    ### Update eZ Enterprise v2.4 to v2.4.2
```
Comment (lserwatka) [inline docs/releases/ez_platform_v2.4.md]: I think this should be changed to v2.4.[0/1] to v2.4.2.

File: docs/releases/ez_platform_v2.4.md [hunk 6]
```diff
@@ -160,3 +160,72 @@ You are now able to load multiple Locations at once. The biggest benefit of this
...
+         token_extractors:
+             authorization_header:
+                 enabled: false
+             cookie:
+                 enabled: false
+             query_parameter:
+                 enabled: false
+    ```
+     
+    By default `HS256` is used as signature algorithm for generated token but we strongly recomend switching to SSH keys. For more information see [`LexikJWTAuthenticationBundle` installation instruction.](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#installation)
```
Comment (DominikaK) [inline docs/releases/ez_platform_v2.4.md]:
```suggestion
    By default `HS256` is used as signature algorithm for generated token but we strongly recommend switching to SSH keys. For more information see [`LexikJWTAuthenticationBundle` installation instruction.](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#installation)
```

**Pull request merged**